test/fixtures/poro.rb

```diff
@@ -42,6 +41,14 @@ def method_missing(meth, *args)
       super
     end
   end
+
+  def cache_key_with_digest
+    "#{cache_key}/#{FILE_DIGEST}"
+  end
+
+  def updated_at
+    @attributes[:updated_at] ||= DateTime.now.to_time
```

Comment:
This should really be `Time.now`.  We can be pretty confident we're using the Gregorian calendar, no?